sprint 8/tema 4.js

```diff
@@ -0,0 +1,38 @@
+
+const vocals = "aeiouAEIOU";
+
+function counterVocals(text) {
+
+    let counter = 0;
+
+    for (let char of text) {     
+        if (vocals.includes(char)) {
+            counter ++;
+        } 
+  }
+  return counter;
+}
+
+text = counterVocals("String de test");
+
+console.log("The number of vowels is: " + text);
+
+
+// let char2 = "String de test";
```

Comment:
aceste comentarii sunt in plus - se poate de sters.

De asemenea - identarea din acest fisier se poate imbunatati